src/thor/bidirectional_astar.cc

```diff
@@ -1053,102 +1069,67 @@ std::vector<std::vector<PathInfo>> BidirectionalAStar::FormPath(GraphReader& gra
     }
 
     // Reverse the list
-    std::reverse(path.begin(), path.end());
-
-    // Special case code if the last edge of the forward path is the destination edge
-    // which means we need to worry about partial distance on the edge
-    if (edgelabels_reverse_[idx2].predecessor() == kInvalidLabel) {
```

Comment:
i know its a bit heavy to use recosting to fix this problem, but it feels really good to be able to remove this anyway :smile: